README.md

```diff
@@ -2,6 +2,9 @@
 
 ## Prerequisites
 - Kubernetes kind/minikuke cluster
+- Cluster must be set up with Ingress controller 
```

Comment:
Do we need this in case of both header based and weight based testing ?

Reply:
Yes with that we don't need to change app endpoint every time  in load-generator/access application for weight based and header based